public/app/plugins/datasource/loki/querybuilder/components/LokiQueryEditorSelector.tsx

```diff
@@ -49,6 +49,13 @@ export const LokiQueryEditorSelector = React.memo<LokiQueryEditorProps>((props)
   const onChangeInternal = (query: LokiQuery) => {
     setDataIsStale(true);
     onChange(query);
+    onRunQuery();
+  };
+
+  const onBlur = () => {
+    if (app !== CoreApp.Explore) {
+      onRunQuery();
+    }
```

Comment:
I would like to keep the behavior the same in Explore & elsewhere

Reply:
I see. Just to be sure about the behaviour in the line-filters input components: 
- onBlur should highlight the `Run Query` button
- onKeyDown with `Enter` will run the query depending on a property from `QueryBuilderOperationParamDef`.